engineyard/ey-cookbooks-dev-v7
# [FBZ-10442] - Migrate Custom Cron Recipe to v7

**Description of your patch**
Migrate Custom Cron Recipe to v7

**Recommended Release Notes**
Added support for custom-cron recipes

**Estimated risk**
Medium

**Components involved**
* `cookbooks/ey-cron/recipes/custom.rb`
* `custom-cookbooks/cron`

**Dependencies**
* `ey-lib`
* `ey-ntp`

**Description of testing done**
* Provision v7 stack environment
* Create custom cron recipe based on `custom-cookbooks/cron/cookbooks/custom-cron`
* Modify `ey-custom` to include `custom-cron`
* Upload to v7 environment
* Trigger an 'Apply' across the environment to execute chef run
* SSH into instance where cronjob is expected to exist
* Execute command `crontab -l` as the user

**QA Instructions**
* Provision v7 stack environment
* Create custom cron recipe based on `custom-cookbooks/cron/cookbooks/custom-cron`
* Modify `ey-custom` to include `custom-cron`
* Upload to v7 environment
* Trigger an 'Apply' across the environment to execute chef run
* SSH into instance where cron job is expected to exist
* Execute command `crontab -l` as the user
* Verify if cron job exists
